### PR TITLE
DESENG-95 Add doc blocks to all areas except services and projects

### PR DESCRIPTION
**Pull request by Baelx**
A follow-up ticket will be created to finish the remaining doc blocks.

### CODE REVIEW

File: src/app/file-upload/file-upload.component.ts [hunk 1]
```diff
@@ -71,8 +71,8 @@ export class FileUploadComponent {
   }
 
   /**
-   * Drag/drop event handler to apply styles when items are dragged/dropped. This gives
-   * the user feedback when using drag/drop in the file picker UI.
+   * Drop event handler to apply styles when items are dragged. This gives
+   * the user feedback when dropping items in the file picker UI.
    *
    * Also add to selected files when they are dropped in the drop area.
    *
```
Comment (jareth-whitney) [inline src/app/file-upload/file-upload.component.ts:78]: Is this one missing a return?

File: src/app/header/header.component.ts [hunk 2]
```diff
@@ -85,24 +91,45 @@ export class HeaderComponent implements OnInit, OnDestroy {
 
   }
 
-  renderMenu(route: String) {
+  /**
+   * Evaluate whether or not to render the admin menu based on the user's roles.
+   * 
+   * @param {string} route
```
Comment (jareth-whitney) [inline src/app/header/header.component.ts]: Is there a description for this param?

**Review by jareth-whitney — CHANGES_REQUESTED** [review]

File: src/app/file-upload/file-upload.component.ts (hunk 1 above)
Comment (Baelx) [reply src/app/file-upload/file-upload.component.ts:78]: No, it's there. The doc block continues past the end of the preview window. You can check the full file if you're not sure.

File: src/app/header/header.component.ts (hunk 2 above)
Comment (Baelx) [reply src/app/header/header.component.ts]: Updated!

**Review by jareth-whitney — APPROVED** [review]
Looks good!

**Pull request merged**